sap_hana/assets/configuration/spec.yaml

```diff
@@ -75,3 +75,8 @@ files:
           type: boolean
       - template: instances/tls
       - template: instances/default
+      - name: tenant_databases_schema
+        description: The schema containing the tenant databases views.
+        value:
+          type: string
```

Comment:
Can we move this field to the top of this file so it's more easily found in the default configuration file and not lost with the generic options? Maybe right underneath `password`?

Reply:
Good idea, done

Reply:
Thanks!